### PR TITLE
Changed display of properties in label if they are too long

### PR DESCRIPTION
**Pull request by ct-anaalbic**
Signed-off-by: ana.albic.comtrade <Ana.Albic@comtrade.com>

Brief description of the PR.
Changed display of name parameter on edit dialogues.

**Related Issue**
This PR fixes issue #2477

**Description of the solution adopted**
If user name on User Edit dialogue, and client id on Device Edit dialogue are too long, than names are resized, and tool tip with full name are added over them. It looks like on screenshots bellow.

**Screenshots**
_UserEditDialog_
image

_DeviceEditDialog_
image

**Any side note on the changes made**
/

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #2477. Can one of the commiters check the code and merge when possible?

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/ActionDialog.java [hunk 1]
```diff
@@ -247,6 +247,26 @@ public void setSubmitButtonState() {
         }
     }
 
+    public String splitTooltipString(String input, int maxLineLength) {
```
Comment (muros-ct) [inline console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/ActionDialog.java]: Please comment the method.
Add parameter validation, what if input is null.
As method is specific provide unit test.
Is this method specific for this class or should be put in utility/common module.
If used only in this class, make it private.

File: console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.app.console.module.api.test;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.eclipse.kapua.app.console.module.api.client.resources.icons.KapuaIcon;
+import org.eclipse.kapua.app.console.module.api.client.ui.dialog.ActionDialog;
+//import org.junit.Test;
```
Comment (muros-ct) [inline console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java]: Sorry can you remove this comment.

File: console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2019 Eurotech and/or its affiliates and others
```
Comment (muros-ct) [inline console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java]: Remove 2016, file is created now.

File: console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+package org.eclipse.kapua.app.console.module.api.test;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.eclipse.kapua.app.console.module.api.client.resources.icons.KapuaIcon;
+import org.eclipse.kapua.app.console.module.api.client.ui.dialog.ActionDialog;
+//import org.junit.Test;
+
+public class SplitStringTest {
+    ActionDialog dialog = new ActionDialog() {
```
Comment (muros-ct) [inline console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java]: This needs to be put into @Before

File: console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        public void submit() {
+        }
+
+        @Override
+        protected void addListeners() {
+        }
+    };
+
+    @Test
+    public void test() {
```
Comment (muros-ct) [inline console/module/api/src/test/java/org/eclipse/kapua/app/console/module/api/test/SplitStringTest.java]: Please test it, it is not about one simple case. Try corner cases.

**Comment by Coduz** [comment]
@ct-anaalbic Please check on the eclipse ECA

**Comment by ct-anaalbic** [comment]
Hi @Coduz 
I will check now.

Regards,
_Ana_

**Comment by muros-ct** [comment]
@ct-anaalbic Your last squashed commit didn't include sign-off.
Amend the commit message with -s for sign-off and to another push for this PR to pass ECA.
